{"name": "test_trigger_test_button", "status": "failed", "statusDetails": {"message": "AssertionError: Failed\nassert 1 == 2", "trace": "def test_trigger_test_button():\n        print(\"Testing Trigger Test Button\")\n    \n>       assert 1==2, \"Failed\"\nE       AssertionError: Failed\nE       assert 1 == 2\n\ntests\\test_stream.py:6: AssertionError"}, "attachments": [{"name": "stdout", "source": "d075c022-a833-4ffd-8699-86fbbad73e48-attachment.txt", "type": "text/plain"}], "start": 1740430555169, "stop": 1740430555169, "uuid": "b5fd9c8b-ad05-490f-b2d8-c2291b631eec", "historyId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "testCaseId": "1f8fbfd19b9570e4ed564d5bd541a9d6", "fullName": "tests.test_stream#test_trigger_test_button", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_stream"}, {"name": "host", "value": "DESKTOP-LUCDFB2"}, {"name": "thread", "value": "3824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_stream"}]}